src/main/java/edu/cornell/kfs/coa/batch/CuCoaBatchConstants.java

```diff
@@ -0,0 +1,29 @@
+package edu.cornell.kfs.coa.batch;
+
+import java.sql.Date;
+
+public class CuCoaBatchConstants {
+    
+    public static final String CSV_FILE_EXTENSION = ".csv";
+    
+    public static class ClosedLaborAccountsFileCreationConstants {
+    
+        public static final int PARAMETER_LABOR_ACCOUNTS_CLOSED_OVER_PAST_DAYS_NOT_SET = -1;
+        public static final int PARAMETER_SET_TO_CREATE_FULL_SEED_FILE = 0;
+        
+        public static final Date SEED_FILE_DEFAULT_FROM_DATE = Date.valueOf("2013-03-01");
```

Comment:
I am wondering if we should move this value to a parameter as well.  The use case I am thinking is workday wants a full extract again, but only back to a certain date.

Reply:
Moving to a parameter.